bower.json

```diff
@@ -5,7 +5,7 @@
         "Zinkerz"
     ],
     "dependencies": {
-        "infra-web-app": "https://github.com/Knowledge-OTP/infra-web-app.git#~v0.0.51"
```

Comment:
I didn't find a pr in infra-web-app from branch #oren/sat-sprint-8"
Please make sure to do it and then we will insert it.